op-node/cmd/genesis/cmd.go

```diff
@@ -24,11 +21,7 @@ import (
 var (
 	l1RPCFlag = &cli.StringFlag{
 		Name:  "l1-rpc",
-		Usage: "RPC URL for an Ethereum L1 node. Cannot be used with --l1-starting-block",
-	}
-	l1StartingBlockFlag = &cli.PathFlag{
-		Name:  "l1-starting-block",
-		Usage: "Path to a JSON file containing the L1 starting block. Overrides the need for using an L1 RPC to fetch the block. Cannot be used with --l1-rpc",
+		Usage: "RPC URL for an Ethereum L1 node",
```

Comment:
You can make this be `Required: true` and then remove the "if is empty, return err" check

Reply:
Good call. Updated here: https://github.com/ethereum-optimism/optimism/pull/11181/commits/d10ad4f28bdb61f690fc6e77a196481b2ed0682f